app/models/garden.rb

```diff
@@ -1,3 +1,7 @@
 class Garden < ApplicationRecord
    has_many :plots
+
+   def plants_with_short_harvest_time
+      Plant.where(plot: plots).where("days_to_harvest < ?", 100).distinct
```

Comment:
This syntax doesn't quite work - in theory, a garden should have many plots, but you are passing a collection of plots to a single `plot` key. Instead, this could start with the garden plots:
`self.plots.joins(:plants).where("days_to_harvest < ?", 100).distinct`